xcdat/regridder/grid.py

```diff
@@ -529,28 +529,30 @@ def create_grid(
     axes = {"x": x, "y": y, "z": z}
     ds = xr.Dataset(attrs={} if attrs is None else attrs.copy())
 
-    for key, item in axes.items():
+    for axis, item in axes.items():
         if item is None:
             continue
 
         if isinstance(item, (tuple, list)):
             if len(item) != 2:
                 raise ValueError(
-                    f"Argument {key!r} should be an xr.DataArray representing "
+                    f"Argument {axis!r} should be an xr.DataArray representing "
```

Comment:
```suggestion
                    f"Argument `{axis}` should be an xr.DataArray representing "
```

Reply:
I don't think we need `!r`.

Reply:
`!r` adds the quotes so you don't need to include them in the f-string.

Reply:
Thanks, that's a cool feature. I briefly Googled it and thought it meant "raw string" to make things like backslash a string.

Reply:
Aw yea `r""` is a raw string, `!r` is just the `repr` of the variable.